tests/idv_service/session/retrieve/allowed.source.response.spec.js

```diff
@@ -0,0 +1,22 @@
+const AllowedSourceResponse = require('../../../../src/idv_service/session/retrieve/allowed.source.response');
+
+describe('AllowedSourceResponse', () => {
+  let extendedAllowedSourceResponse;
+
+  beforeEach(() => {
+    class ExtendedAllowedSourceResponse extends AllowedSourceResponse {
+      constructor(type) {
+        super();
+        this.type = type;
+      }
```

Comment:
No need to defined the constructor, that will be inherited, and that is exactly what we want to test here, ie, does the implementation in the abstract doing the right thing, so it would be counter productive to have an implementation in the sub class.